pkg/storage/edv/edvstorage.go

```diff
@@ -150,7 +150,7 @@ func (c *Config) createEncryptedFormatter(ctx context.Context,
 
 	startNewJWEEncrypt := time.Now()
 
-	encrypter, err := jose.NewJWEEncrypt(encAlg, encType, "", nil, []*crypto.PublicKey{pubKey}, c.CryptoService)
+	encrypter, err := jose.NewJWEEncrypt(encAlg, encType, "", "", nil, []*crypto.PublicKey{pubKey}, c.CryptoService)
```

Comment:
@DRK3 do you want to add a ctyType, like encType for edv storage?

Reply:
Not sure - what impact does it have? @Baha-sk

Reply:
it can be empty, but `encType` and `cty` must remain the same for encryption/decryption to work. 

encType represent's the envelope media type
cty represent's the payload's media type

we can either use `application/json` or keep them empty, but once set, they cannot change.